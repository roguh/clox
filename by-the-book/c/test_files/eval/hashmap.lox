print({1:2, 'abc': 'def', "\"land's end\"": 3});
var hm = {'a':12,12:'a'}; print(hm['a']); print(hm[12]);
var nested = {12:{12:{12:[{12:[13,14]},15]}}};
print(nested[12]);
print(nested[12][12]);
print(nested[12][12][12][0]);
print(nested[12][12][12][0][12][1]);
