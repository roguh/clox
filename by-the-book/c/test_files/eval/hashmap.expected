{"\\"land's end\\"": 3, "abc": "def", 1: 2}
12
a
{12: {12: [{12: [13, 14]}, 15]}}
{12: [{12: [13, 14]}, 15]}
{12: [13, 14]}
14
